### ISSUE TITLE
The Broccoli Plugin: [SourceMapConcat: Concat: Vendor /assets/vendor.js] failed

### ISSUE THREAD

**Issue by joeczucha** [kalcifer/ember-dragula#15]
Using the latest version of ember-cli (2.4.2), a clean install on a newly generated project. When attempting to run `ember serve` I get the following:

```
Serving on http://localhost:4200/
The Broccoli Plugin: [SourceMapConcat: Concat: Vendor /assets/vendor.js] failed with:
Error: ENOENT: no such file or directory, stat '/Users/joe/Sites/labs/dragular/tmp/source_map_concat-input_base_path-OnNb5kIh.tmp/0/bower_components/dragula.js/dist/dragula.js'
    at Error (native)
    at Object.fs.statSync (fs.js:892:18)
    at ConcatWithMaps.keyForFile (/Users/joe/Sites/labs/dragular/node_modules/ember-cli/node_modules/broccoli-concat/node_modules/broccoli-caching-writer/index.js:90:20)
    at Array.map (native)
    at ConcatWithMaps.CachingWriter._conditionalBuild (/Users/joe/Sites/labs/dragular/node_modules/ember-cli/node_modules/broccoli-concat/node_modules/broccoli-caching-writer/index.js:112:65)
    at /Users/joe/Sites/labs/dragular/node_modules/ember-cli/node_modules/broccoli-concat/node_modules/broccoli-caching-writer/node_modules/broccoli-plugin/read_compat.js:61:34
    at lib$rsvp$$internal$$tryCatch (/Users/joe/Sites/labs/dragular/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:1036:16)
    at lib$rsvp$$internal$$invokeCallback (/Users/joe/Sites/labs/dragular/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:1048:17)
    at lib$rsvp$$internal$$publish (/Users/joe/Sites/labs/dragular/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:1019:11)
    at lib$rsvp$asap$$flush (/Users/joe/Sites/labs/dragular/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:1198:9)

The broccoli plugin was instantiated at:
    at ConcatWithMaps.Plugin (/Users/joe/Sites/labs/dragular/node_modules/ember-cli/node_modules/broccoli-concat/node_modules/broccoli-caching-writer/node_modules/broccoli-plugin/index.js:10:31)
    at ConcatWithMaps.CachingWriter [as constructor] (/Users/joe/Sites/labs/dragular/node_modules/ember-cli/node_modules/broccoli-concat/node_modules/broccoli-caching-writer/index.js:21:10)
    at new ConcatWithMaps (/Users/joe/Sites/labs/dragular/node_modules/ember-cli/node_modules/broccoli-concat/concat.js:22:17)
    at module.exports (/Users/joe/Sites/labs/dragular/node_modules/ember-cli/node_modules/broccoli-concat/index.js:26:10)
    at EmberApp.concatFiles (/Users/joe/Sites/labs/dragular/node_modules/ember-cli/lib/broccoli/ember-app.js:329:10)
    at EmberApp.javascript (/Users/joe/Sites/labs/dragular/node_modules/ember-cli/lib/broccoli/ember-app.js:1190:12)
    at EmberApp.toArray (/Users/joe/Sites/labs/dragular/node_modules/ember-cli/lib/broccoli/ember-app.js:1542:10)
    at EmberApp.toTree (/Users/joe/Sites/labs/dragular/node_modules/ember-cli/lib/broccoli/ember-app.js:1564:30)
    at module.exports (/Users/joe/Sites/labs/dragular/ember-cli-build.js:23:14)
    at Class.module.exports.Task.extend.setupBroccoliBuilder (/Users/joe/Sites/labs/dragular/node_modules/ember-cli/lib/models/builder.js:55:19)
```

Thanks in advance, and for all your work so far.

**Comment by VincentMolinie** [comment]
Just checkout version 1.9.1, I already ask to change it. It's just the path that is wrong.

**Comment by pksjce** [comment]
Hi
@joeczucha - Hey I will fix this up by this weekend. Meanwhile can you use the old version?

**Comment by andrei-sheina** [comment]
Have the same issue

**Comment by danilovaz** [comment]
The same issue here. :disappointed: 

I'm using ember 2.5 with ember-cli 2.4.3

**Comment by VincentMolinie** [comment]
I already gave the solution ! Just checkout to the older version.
If you really want this version you have to fix it manually: 
Go to node_modules/ember-dragula/ember-cli-build.js and replace all dragula.js/ by dragula/ and same thing in node_modules/ember-dragula/index.js

**Comment by danilovaz** [comment]
Ok, it's works for now :smile: thanks

**Comment by pksjce** [comment]
Sorry for the delay. I have merged the concerned PR and fixed some bower dependencies. I have now published 1.9.3. Please update to this version.
